webextensions/api/webNavigation.json

```diff
@@ -29,7 +29,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webNavigation/TransitionQualifier",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "44"
```

Comment:
✅  https://github.com/chromium/chromium/commit/00f1fc22bfcc653b47d62778170af3eee9855720 in [44.0.2359.0](https://github.com/chromium/chromium/releases/tag/44.0.2359.0) before 44.0.2403.